scripts/anonymize_database.rb

```diff
@@ -1,7 +1,7 @@
 Anonymizer.anonymize_all_data!
 
 # Sanity checks
-if User.where.not(first_name: "[valeur anonymisée]").any?
+if User.where.not(last_name: "[valeur anonymisée]").any?
```

Comment:
> three users have a blank first name in production data

J'en compte 0, tu as fait comment pour en trouver 3 ?

Et si on ajoutait une contrainte `NOT NULL` sur ces colonnes qui sont en réalité toujours remplies ? Encore une fois, pour faire coller nos contraintes de DB à la réalité de nos données. Comme ça on ne se posera plus la question "Est-ce qu'il y a des usagers sans nom ?", on aura juste à regarder le schema !

Reply:
un simple `User.where(first_name: "").count` (id 98718, 44581, 208392)

Reply:
Haha, j'avais lancé `User.where(first_name: nil).count` ! Je redouble donc mon commentaire sur l'ajout d'un `NOT NULL` sur ces colonnes, qu'on soit au clair !